lib/themes/dosomething/paraneue_dosomething/js/components/Button.js

```diff
@@ -0,0 +1,46 @@
+const React = require('react');
+
+/**
+ * Button Component
+ * <Button />
+ */
+const Button = (props) => {
+    const classes = ['button'].concat(props.classes);
+
+    if (props.loading === true) {
+      classes.push('is-loading');
+    }
+
+    if (props.success === true) {
+      classes.push('is-successful');
+    }
+
+    if (props.type) {
+      classes.push(props.type);
+    }
+
+    if (props.reactive) {
+      classes.push('-reactive');
+    }
+
+    return (
+      <button className={classes.join(' ')} onClick={props.onClick} disabled={props.disabled}>{props.children}</button>
+    );
+}
+
+Button.defaultProps = {
+  disabled: false,
+  loading: false,
+  success: false,
+  children: 'Submit',
+};
+
+Button.propTypes = {
+  disabled: React.PropTypes.bool,
+  loading: React.PropTypes.bool,
+  onClick: React.PropTypes.func.isRequired,
+  reactive: React.PropTypes.bool,
+  type: React.PropTypes.oneOf(['-secondary', '-tertiary']),
```

Comment:
This is such a nitpick, but it'd be cool to have the type be `secondary` or `tertiary`, and then add the `-` above:

```js
    if (props.type) {
       classes.push('-' + props.type);
    }
```